internal/backends/android-activity/build.rs

```diff
@@ -27,16 +27,14 @@ fn main() {
     let classpath = find_latest_version(android_home.join("platforms"), "android.jar")
         .expect("No Android platforms found");
 
-    // Try to locate javac
-    let javac_path = match env_var("JAVA_HOME") {
-        Ok(val) => {
-            if cfg!(windows) {
-                format!("{}\\bin\\javac.exe", val)
-            } else {
-                format!("{}/bin/javac", val)
-            }
-        }
-        Err(_) => String::from("javac"),
+    // Try to locate javac and java
+    let javac_java = env_var("JAVA_HOME")
+        .map(|home| PathBuf::from(home).join("bin"))
+        .map(|bin| (bin.join("javac"), bin.join("java")));
```

Comment:
I hope this also work on windows without the .exe

Reply:
Yes, I have tested it, it works on Windows 10 indeed.

Reply:
I did another test under a non-ASCII path with a space and it works too.